paas/install_client.go

```diff
@@ -132,15 +155,14 @@ func (c *InstallClient) Uninstall(cmd *cobra.Command) error {
 
 	for _, deployment := range []kubernetes.Deployment{
 		&deployments.Workloads{Timeout: DefaultTimeoutSec},
+		&deployments.ServiceCatalog{Timeout: DefaultTimeoutSec},
 		&deployments.Tekton{Timeout: DefaultTimeoutSec},
 		&deployments.Registry{Timeout: DefaultTimeoutSec},
 		&deployments.Gitea{Timeout: DefaultTimeoutSec},
 		&deployments.Quarks{Timeout: DefaultTimeoutSec},
 		&deployments.Traefik{Timeout: DefaultTimeoutSec},
 	} {
-		details.Info("remove", "Deployment", deployment.ID())
-		err := deployment.Delete(c.kubeClient, c.ui)
-		if err != nil {
+		if err := c.UninstallDeployment(deployment, details); err != nil {
```

Comment:
Uninstalling carrier with `in-cluster services` enabled leaves the relevant pod, i.e. `minibroker` active and running.
Pretty sure that is a general thing for the various services deployments.
Should possible add them here, unconditionally.
Anything not enabled will be skipped.

Reply:
Good catch! I will add them to the Uninstall of  `carrier uninstall`

Reply:
Done